README.md

```diff
@@ -33,6 +33,14 @@ there is an explanation about stable packages and MELPA and
 
 `m-x package-install helm-dash RET`
 
+
+## Installing docsets
+
+Helm-dash uses the same docsets as [Dash](http://www.kapeli.com/dash).
+You can install them with `m-x helm-dash-install-docset` for the
+official docsets or `m-x helm-dash-install-user-docset` for user
+contributed docsets (experimental).
```

Comment:
Could you add a reference to new function `helm-dash-install-docset-from-file` ?